RecoEgamma/EgammaTools/interface/EgammaPCAHelper.h

```diff
@@ -0,0 +1,115 @@
+//--------------------------------------------------------------------------------------------------
+//
+// EGammaPCAHelper
+//
+// Helper Class to compute PCA
+//
+//
+//--------------------------------------------------------------------------------------------------
+#ifndef EGammaPCAHelper_H
+#define EGammaPCAHelper_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h